example_setups/librispeech/ctc_rnnt_standalone_2024/experiments/rnnt_bpe/baseline.py

```diff
@@ -0,0 +1,243 @@
+
+    train_settings = DatasetSettings(
+        preemphasis=0.97,  # TODO: Check if this is really useful
+        peak_normalization=True,  # TODO: Also check if really useful, older Attention setups did not have that
+        # training
+        train_partition_epoch=10,
+        train_seq_ordering="laplace:.1000",
+    )
+
+    # build the training datasets object containing train, cv, dev-train and the extern_data dict
+    train_data_bpe5000 = build_bpe_training_datasets(
+        prefix=prefix_name,
+        librispeech_key="train-other-960",
+        bpe_size=5000,
+        settings=train_settings,
+        use_postfix=True,  # RNN-T now, use postfix
+    )
+    label_datastream_bpe5000 = cast(LabelDatastream, train_data_bpe5000.datastreams["labels"])
+    vocab_size_without_blank = label_datastream_bpe5000.vocab_size
+
+    dev_dataset_tuples = {}
+    for testset in ["dev-clean", "dev-other"]:
+        dev_dataset_tuples[testset] = build_test_dataset(
+            dataset_key=testset,
+            settings=train_settings,
+        )
+
+    test_dataset_tuples = {}
+    for testset in ["test-clean", "test-other"]:
+        test_dataset_tuples[testset] = build_test_dataset(
+            dataset_key=testset,
+            settings=train_settings,
+        )
+
+    default_returnn = {
+        "returnn_exe": RETURNN_EXE,
+        "returnn_root": MINI_RETURNN_ROOT,
+    }
+
+    from ...pytorch_networks.rnnt.decoder.experimental_rnnt_decoder import DecoderConfig
+
+    def evaluate_helper(
+        training_name: str,
+        asr_model: ASRModel,
+        base_decoder_config: DecoderConfig,
+        beam_size: int = 1,
+    ):
+        """
+        Example helper to execute tuning over lm_scales and prior scales.
+        With the best values runs test-clean and test-other.
+
+        This is just a reference helper and can (should) be freely changed, copied, modified etc...
+
+        :param training_name: for alias and output names
+        :param asr_model: ASR model to use
+        :param base_decoder_config: any decoder config dataclass
+
+        """
+        decoder_config = copy.deepcopy(base_decoder_config)
+        decoder_config.beam_size = beam_size
+        search_name = training_name + "/search_bs%i" % beam_size
+        search_jobs, wers = search(
+            search_name,
+            forward_config={},
+            asr_model=asr_model,
+            decoder_module="rnnt.decoder.experimental_rnnt_decoder",
+            decoder_args={"config": asdict(decoder_config)},
+            test_dataset_tuples={**dev_dataset_tuples, **test_dataset_tuples},
+            **default_returnn,
+        )
+
+    greedy_decoder_config_bpe5000 = DecoderConfig(
+        beam_size=1, returnn_vocab=label_datastream_bpe5000.vocab  # greedy as default
+    )
+    bs10_decoder_config_bpe5000 = DecoderConfig(
+        beam_size=1, returnn_vocab=label_datastream_bpe5000.vocab  # greedy as default
```

Comment:
```suggestion
        beam_size=10, returnn_vocab=label_datastream_bpe5000.vocab
```
Shouldn't this be 10 then? Or what does bs10 mean?